### ISSUE TITLE
The help document of php misses two options

### ISSUE THREAD

**Issue by ccccmd** [php/doc-en#1863]
The "-h" option of php misses the option "C", "q".

+  For the option "C", the following code appears during option parsing in sapi/cli/php_cli.c:701-703 for version 8.0.23.

```
			case 'C': /* don't chdir to the script directory */
				/* This is default so NOP */
				break;
```

+  For the option "q", the following code appears during option parsing in sapi/cli/php_cli.c:737-739 for version 8.0.23.

```
			case 'q': /* do not generate HTTP headers */
				/* This is default so NOP */
				break;
```

Although it has no actual function, it is still possible forget to modify the help document after adding the function in a subsequent update.

But they do not appear in the document provided by "-h".

It may prevent users from using the relevant function.

**Comment by cmb69** [comment]
Right, but again this is a php-src issue (the options are already [documented in the PHP manual](https://www.php.net/manual/en/features.commandline.options.php)). And while I agree that these options should be listed by `php -h`, we should consider to remove them from the CLI API.
